server.py

```diff
@@ -0,0 +1,149 @@
+import json
+from six.moves.urllib.request import urlopen
+from functools import wraps
+
+from flask import Flask, request, jsonify, _request_ctx_stack
+from flask_cors import cross_origin
+from jose import jwt
+
+app = Flask(__name__)
+
+with open('config.json') as json_data_file:
```

Comment:
It'll be nice to open config files from URL than system dependency (or) if it is meant to be packaged along with the app, README can talk more about the contents of this config file.